kubernetes/client/rest.py

```diff
@@ -157,7 +157,8 @@ def request(self, method, url, query_params=None, headers=None,
             if method in ['POST', 'PUT', 'PATCH', 'OPTIONS', 'DELETE']:
                 if query_params:
                     url += '?' + urlencode(query_params)
-                if re.search('json', headers['Content-Type'], re.IGNORECASE):
+                if (re.search('json', headers['Content-Type'], re.IGNORECASE) or
```

Comment:
Unfortunately this file is generated by the OpenAPI generator, I'm not sure what the process is for carrying patches to the generated files but usually these need to be fixed in the generator itself

@roycaihw what should we do here?